packages/graphql-hooks/src/GraphQLClient.js

```diff
@@ -62,6 +58,10 @@ class GraphQLClient {
 
     const error = result.error
 
+    if (!error) {
```

Comment:
@panz3r If we return here, we'd have never called the final `console.groupEnd`, meaning the all further console logs will be nested. This happened before https://github.com/nearform/graphql-hooks/issues/85.

An alternative would be to add check in each `if ` below:

```js
if (error && error.fetchError) {...}
if (error && error.httpError) {...}
if (error && error.graphQLErrors && error.graphQLErrors.length > 0) {...}
```

Reply:
To avoid repeating the issue #85, I improved the tests around `logErrorResult` method and updated the skip implementation.